packages/webview_flutter/example/test_driver/webview.dart

```diff
@@ -0,0 +1,57 @@
+// Copyright 2019, the Chromium project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:flutter/widgets.dart';
+import 'package:flutter_driver/driver_extension.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:webview_flutter/webview_flutter.dart';
+
+void main() {
+  final Completer<String> allTestsCompleter = Completer<String>();
+  enableFlutterDriverExtension(handler: (_) => allTestsCompleter.future);
+  tearDownAll(() => allTestsCompleter.complete(null));
+
+  test('initalUrl', () async {
+    final Completer<WebViewController> controllerCompleter =
+        Completer<WebViewController>();
+    runApp(
```

Comment:
Consider moving into an async helper function to avoid code duplication with the test below.

Reply:
I typically prefer to have code duplication across tests vs. extracting "repetitive" stuff from test cases to a common method so when a test fail and you're coming to look at it, you can tell right away what the test is doing without having to go and understand the what the common methods are doing.

Curious what are your thoughts about this?

Reply:
I don’t feel strongly about it and I could go either way. It feels somewhat specific to the way that this plugin initializes itself so I wouldn’t generalize this decision to other plugins. If you like it better with repetition then let’s leave it.